src/current/v22.2/use-cloud-storage.md

```diff
@@ -253,9 +253,17 @@ The following S3 connection URI uses the `INTELLIGENT_TIERING` storage class:
 
 While Cockroach Labs supports configuring an AWS storage class, we only test against S3 Standard. We recommend implementing your own testing with other storage classes.
 
-{{site.data.alerts.callout_info}}
-[Incremental backups](take-full-and-incremental-backups.html#incremental-backups) are **not** compatible with the S3 Glacier Flexible Retrieval or Glacier Deep Archive storage classes. Incremental backups require ad-hoc reading of previous backups, which is not possible with the Glacier Flexible Retrieval or Glacier Deep Archive storage classes as they do not allow immediate access to S3 objects without first restoring the objects. See Amazon's documentation on [Restoring an archived object](https://docs.aws.amazon.com/AmazonS3/latest/userguide/restoring-objects.html) for more detail.
-{{site.data.alerts.end}}
+#### Incremental backups and archive storage classes
```

Comment:
Is this correct? Although we say not compatible... It seems like a user can still use Glacier, but not when "actively" taking backups on that chain. So, take backup --> take incrementals --> stop backups --> archive objects into Glacier --> restore ALL objects back to S3 --> successful CRDB restore. Hopefully the few paragraphs here capture that.

Reply:
Perfect, thank you!